templates/rhpam71-trial-ephemeral.yaml

```diff
@@ -66,6 +65,11 @@ parameters:
   name: KIE_SERVER_CONTROLLER_USER
   value: controllerUser
   required: false
+- displayName: KIE Server Controller Protocol
+  description: KIE server controller protocol (Used to set the org.kie.server.controller system property)
```

Comment:
upper case?

Reply:
@spolti in all the other controller parameters, the displayName uses upper case, but the description uses lower case. I was just being consistent...